vignettes/Vignette_Infillcrits.Rmd

```diff
@@ -0,0 +1,112 @@
+---
+title: "mlrMBO: Infill criteria"
+output: html_document
```

Comment:
Can you follow the header and the structure of the other vignettes please?
For running text I strongly favor a line break after each sentence.